tableauserverclient/server/endpoint/views_endpoint.py

```diff
@@ -1,20 +1,31 @@
 from .endpoint import Endpoint
 from .exceptions import MissingRequiredFieldError
-from .. import ViewItem, PaginationItem
+from .resource_tagger import _ResourceTagger
+from .. import RequestFactory, ViewItem, PaginationItem
+from ...models.tag_item import TagItem
 import logging
+import copy
 
 logger = logging.getLogger('tableau.endpoint.views')
 
 
 class Views(Endpoint):
+    def __init__(self, parent_srv):
+        super(Views, self).__init__(parent_srv)
+        self._resource_tagger = _ResourceTagger(parent_srv)
+
+    # Used because populate_preview_image functionaliy requires workbook endpoint
     @property
-    def baseurl(self):
+    def siteurl(self):
```

Comment:
Oh, now I feel like this should move to the Endpoint object.  Is siteurl the same everywhere?